Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -245,6 +245,8 @@ theorem isBasis_basicOpen (X : Scheme) [IsAffine X] :
 
 namespace IsAffineOpen
 
+-- Adaptation note: 2024-04-23
+-- TODO check if this `.{u}` restricts anything
 variable {X Y : Scheme.{u}} {U : Opens X} (hU : IsAffineOpen U) (f : X.presheaf.obj (op U))
```

Comment:
I guess this can now be removed, assuming this was checked in review?

Reply:
Done.

Reply:
Do we need to keep this as an adaptation note? These were already quite slow before.

Reply:
Given that it was already changed on master, let's say no? I'm not exactly sure what we would add.